kotlin-spark-api.json

```diff
@@ -0,0 +1,30 @@
+{
+  "description": "Kotlin for Apache® Spark™",
+  "properties": {
+    "spark": "3.2",
+    "v": "2.0.0"
+  },
+  "link": "https://github.com/JetBrains/kotlin-spark-api",
+  "dependencies": [
+    "org.jetbrains.kotlinx.spark:kotlin-spark-api-$spark:$v"
+  ],
+  "init": [
+    "%dumpClassesForSpark",
```

Comment:
Same question: is it something for debug?

Reply:
oh these files need to be deleted :) They will be in the official kotlin jupyter libraries repo

Reply:
plus the code in there is now in the integration class itself